README.md

```diff
@@ -20,45 +20,44 @@ Project
  * Various fixes
  * Overworld Glitches Logic
  * Newer Entrance Randomizer Logic, allowing more potential item and boss locations
- * completely redesigned command interface, with `!help` and `/help`
- * New Goal: local triforce hunt - limits triforce pieces to your own world so it is your own goal to accomplish
+ * New Goal: local triforce hunt - Keeps triforce pieces local to your world
  
 MultiMystery.py
  * Allows you to generate a Multiworld with individual player mystery weights. Since weights can also be set to 100%, this also allows for individual settings for each player in a regular multiworld.
-Basis is a .yaml file that sets these weights. You can find an [easy.yaml](https://github.com/Berserker66/MultiWorld-Utilities/blob/master/easy.yaml) in this project folder to get started.
```

Comment:
I forgot to point this out in my review. I would suggest keeping this text as ".yaml" or even ".YAML". The dot isn't redundant in my opinion. This prevents a misinterpretation of the file type as being anything other than a specific file format, even for those systems without typical file extensions.

Reply:
Address in commit https://github.com/Berserker66/MultiWorld-Utilities/pull/133/commits/dd2694ff1c8f6b097ea4708ed368958c37e7fd63 .

For some reason I kept reading it out loud in my head as dot yaml and it sounded odd but I agree that it does prevent confusion if its explicitly called out as .yaml.